embedded/EscapeROOM/radio_morse_panel/radio_morse_panel.c

```diff
@@ -243,7 +214,8 @@ ISR(PCINT_vect)
 		play_flag = MAX_REPEAT;
```

Comment:
is it a flag or index or repeat count? Flag usually implies bit in variable or boolean type variable.